src/main/java/org/javacs/config/MavenConfig.java

```diff
@@ -0,0 +1,129 @@
+package org.javacs.config;
```

Comment:
This is the same case as `BazelConfig`, except this is still not being used. Couldn't really think on the better logic for picking this one over `InferConfig`, so I decided not to hook it on this PR.